cfn-resources/cloud-backup-snapshot/docs/README.md

```diff
@@ -135,6 +141,18 @@ Unique string that identifies the Amazon Web Services (AWS) Key Management Servi
 
 List of returned documents that MongoDB Cloud provides when completing this request.
 
+#### ItemsPerPage
+
+Number of items that the response returns per page.
```

Comment:
Noticed this detail was removed for these properties

Any reason? Or should they be reintroduced?

```.md
_Required_: No

_Type_: Integer

_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```

Reply:
These are the properties related to pagination list .We are not using these properties in the CFN resource.They accidentally got introduced through schema auto generation script. So we removed them